puppetboard/templates/layout.html

```diff
@@ -56,6 +57,64 @@
           $.getScript('{{url_for('static', filename='js/tables.js')}}')
           {% block onload_script %} {% endblock onload_script %}
         })
+       
+      $('#export_data_button').click(function() {
+        var titles = [];
+        var data = [];
+
+        $('.dataTable th').each(function() {
```

Comment:
The table doesn't have this class name on any of the pages I'm looking at.

What page did you test this on?

Reply:
Ok I see it works perfect on the Inventory page.

Do you only intend to use it with inventory? Or do you want it to work on other pages too?

Reply:
No, it works for all pages. I have tested on all facts, and it works there too

Reply:
What is the browser? I have tested with IE and sometimes it doesn't work there

Reply:
It seems to work on all fact pages for me too.

It doesn't work on Overview, Nodes, Facts, Reports, Metrics, Catalogs (kind of works), or Query.

Reply:
This is because the data in Nodes,Facts ( try opening a fact, e.g. "ip adresses" there it works), reports is not initialized from dataTable, so there are no headers and data cells. So this code will only work on any data from dataTable